esileapclient/osc/v1/offer.py

```diff
@@ -40,6 +40,11 @@ def get_parser(self, prog_name):
             required=False,
             help="Time when the offer will expire and no longer be "
                  "'available'.")
+        parser.add_argument(
+            '--lessee-id',
+            dest='lessee_id',
+            required=False,
+            help="Project subtree to which this offer will be limited.")
```

Comment:
Doesnt the lessee_id also allow parent projects to claim too? So its anyone on the tree going up as well. Just slightly inaccurate language i think

Reply:
Oh, it actually doesn't (I think); we match by taking the requesting project's ID, going up the tree, and seeing if any of those IDs match the offer's lessee ID. A parent project of the offer's lessee ID shouldn't match as a result.